src/Tizen.NUI/src/internal/WebView/WebBackForwardList.cs

```diff
@@ -50,23 +57,66 @@ public string Title
         {
             get
             {
-                return Interop.WebBackForwardListItem.GetTitle(SwigCPtr);
+                string ret = Interop.WebBackForwardListItem.GetTitle(SwigCPtr);
+                if (NDalicPINVOKE.SWIGPendingException.Pending) return null;
+                return ret;
             }
         }
 
         /// <summary>
         /// Get original url.
         /// </summary>
         [EditorBrowsable(EditorBrowsableState.Never)]
-        public Uri OriginalUrl
+        public string OriginalUrl
         {
             get
             {
-                return new Uri(Interop.WebBackForwardListItem.GetOriginalUrl(SwigCPtr));
+                string ret = Interop.WebBackForwardListItem.GetOriginalUrl(SwigCPtr);
+                if (NDalicPINVOKE.SWIGPendingException.Pending) return null;
+                return ret;
             }
         }
     }
 
+    /// <summary>
+    /// WebBackForwardCopiedListItems is a class for back-forward copied list item of web view.
+    /// </summary>
+    [EditorBrowsable(EditorBrowsableState.Never)]
+    public class WebBackForwardCopiedItemList : Disposable
+    {
+        internal WebBackForwardCopiedItemList(global::System.IntPtr cPtr, bool cMemoryOwn) : base(cPtr, cMemoryOwn)
+        {
+        }
+
+        protected override void ReleaseSwigCPtr(System.Runtime.InteropServices.HandleRef swigCPtr)
+        {
+            Interop.WebBackForwardCopiedItemList.DeleteCopiedItems(swigCPtr);
+        }
+
+        /// <summary>
+        /// Get the count of copied items.
+        /// </summary>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public uint GetCopiedItemsCount()
+        {
+            uint ret = Interop.WebBackForwardCopiedItemList.GetCopiedItemsCount(SwigCPtr);
+            if (NDalicPINVOKE.SWIGPendingException.Pending) return 0;
+            return ret;
+        }
+
+        /// <summary>
+        /// Get the item according to index
+        /// </summary>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public WebBackForwardListItem GetItemAtIndexFromCopiedItems(uint index)
```

Comment:
I think `GetItemAtIndex` is enough. What do you think?

Reply:
> I think `GetItemAtIndex` is enough. What do you think?

I agree with you:)
Fixed